### ISSUE TITLE
Specify a device for kernel execution

### ISSUE THREAD

**Issue by zijiecong** [aparapi/aparapi#27]
Doc [Choosing Specific Devices for Execution](https://github.com/aparapi/aparapi/blob/master/doc/ChoosingSpecificDevicesForExecution.md) points out that one can choose a specific device for kernel execution. 

However it seems that Kernel.setExecutionMode() is deprecated and aparapi is unable to automatically set the execution mode for a specific device.

**Comment by freemo** [comment]
This issue was moved to Syncleus/aparapi#11

**Comment by rlamothe** [comment]
Yes, this has changed in a recent commit by another contributor. There are a number of outstanding documentation tasks that need to be re-visited, I will add this one to the pile. Thanks!
